File: dayTwo/python/book_suggestion_app.py
import random
book_rack = ["Learn Python in 12 hrs", "Everything is F*ucked", "A Moving man", "Delusion or Confidence", "Take Take and Take Some More"]

def book_pages():
    pages = [];

    for count in range (1, 100+1):
        pages.append(count)
    return pages;
    

def suggest_books():
    if(book_rack == []):
        return f"No Sugestion!\nBook Rack is Empty!\n";

    suggested_book = random.choice(book_rack)
    return suggested_book

def pages():
    number_of_pages = book_pages()

    page = random.choice(number_of_pages)
    if(book_rack == []):
        return f"Nil\n";
    return page;


def suggest_book_and_page():
    page = pages()
    book_title = suggest_books()
    return f"Book Title: {book_title}\nPage: {page}";

def add_books(title):
    if (title not in book_rack):
        return f"Book Added Successfully\n";
    return f"Book already added\n";

def remove_books(title):
    if(title not in book_rack):
        return f"Book not found!\n";
    book_rack.remove(title)
    return f"Book removed Successfully!\n";


def update_books(old_title, new_title):
    if(old_title in book_rack):
        book_rack.remove(old_title)
        book_rack.append(new_title)
        return f"Book updated Successfully!\n";
    return f"Book not Found!\n";

def show_all_books():
    if(book_rack == []):
        return f"No Book(s) in Rack 🥲️";
    return book_rack;

#print(update_books("A Moving man", "Man Artica"))
#print(remove_books("Learn Python in 12 hrs"))
#print(remove_books("Everything is F*ucked"))
#print(remove_books("A Moving man"))
#print(remove_books("Delusion or Confidence"))
#print(remove_books("Take Take and Take Some More"))
#print(show_all_books())
#print(suggest_book_and_page())

listss = [4,5,6,7,8,9]
print(listss[-1])

File: dayTwo/javaScript/book_suggestion_app.js
//import random
let bookRack = ["JavScript How to be a Programmer", "Things Fall Apart", "The Glutton", "The Pain Of Knowing A Fool", "The Area Mama", "Happy Sad Happy Sad"];

function book_pages(){
    let pages = []
    let count = 0
    let numbers = 0
    for(; count < 100;){
        pages[count++] = ++numbers
    }
    return pages;
}   

function suggest_books(){
    if(bookRack.length === 0)
        return "No Sugestion!\nBook Rack is Empty!\n";

    let suggestedBook = Math.floor(Math.random() * bookRack.length)
    return bookRack[suggestedBook];
}

function pages(){
    let number_of_pages = book_pages()

    let page = Math.floor(Math.random() * number_of_pages.length)
    if(bookRack.length === 0){
        return "Nil\n";
    }
    return number_of_pages[page];
}

function suggest_book_and_page(){
    let page = pages()
    let book_title = suggest_books()
    return `\nBook Title: ${book_title}\nPage: ${page}`;
}

function add_books(title){
    if (bookRack.includes(title)){
        return "Book already added!\n";
    }
    else{
        bookRack.push(title)
        return "Book Added Successfully!\n";
    }
}

function remove_books(title){
    if(bookRack.includes(title)){
        bookRack.splice(bookRack.indexOf(title), 1)
        return "Book removed Successfully!\n";
    }
    return "Book not found!\n";
}

function update_books(oldTitle, newTitle){
    if(bookRack.includes(oldTitle)){
        bookRack.splice(bookRack.indexOf(oldTitle), 1)
        bookRack.push(newTitle)
        return "Book updated Successfully!\n";
    }
    return "Book not Found!\n";
}

function show_all_books(){
    if(bookRack.length === 0)
        return "No Book(s) in Rack 🥲️";
    return bookRack;
}

module.exports = {pages, bookRack, suggest_books, suggest_book_and_page, add_books, remove_books, update_books, show_all_books};


